onnxruntime/core/providers/cpu/math/matmul.h

```diff
@@ -15,6 +15,14 @@ class MatMul final : public OpKernel {
   Status Compute(OpKernelContext* context) const override;
 };
 
+static inline bool ort_use_fastmath_mode() {
```

Comment:
could you please replace the env with a session option?

Reply:
thanks, @yufenglee , will replace.